### DISCUSSION TITLE
How to apply nitro for main account only

### DISCUSSION THREAD

**Discussion by NEON-CYC** [Vedza/NitroSniperGo#152] [Q&A]
How do I use my alt account to snipe nitro and apply them to my main ? Im very new to this

**Comment by Vedza** [comment]
Hi,
This is already what the sniper does 👍

**Reply by NEON-CYC** [reply]
How often do you get a drop ?

**Reply by Vedza** [reply]
It depends on your servers, it can take a few hours like a few days or week
